test/Microsoft.Health.SqlServer.Web/Features/Schema/Migrations/2.sql

```diff
@@ -6,6 +6,32 @@
 
 INSERT INTO dbo.SchemaVersion
 VALUES
-    (2, 'started')
+    (3, 'started')
```

Comment:
Is it an intentional change to insert '3' in 2.sql or I might be missing something here.
I know it is little overhead and am sorry to not catch up on the user story to automate this as it got deprioritized on my board.

Reply:
No that was not intended :) thanks!